api/src/components/user/service.ts

```diff
@@ -130,17 +130,39 @@ export const get = async (id: string, isAccountOwner = false) => {
     return user;
 };
 
+export const getCoAuthoredPublications = async (id: string) => {
+    const coAuthoredPublications = await client.prisma.coAuthors.findMany({
+        select: {
+            publicationId: true,
+            linkedUser: true
+        },
+        where: {
+            linkedUser: id
+        }
+    })
+
+    return coAuthoredPublications;
+}
+
 export const getPublications = async (id: string, params: I.UserPublicationsFilters, isAccountOwner: boolean) => {
     const { offset, limit, orderBy, orderDirection } = params;
 
     // Account owners can retrieve their DRAFT publications also
     const statuses: Array<I.ValidStatuses> = isAccountOwner ? ['DRAFT', 'LIVE'] : ['LIVE'];
 
+    // Get id's of publications the user is co author of
+    const coAuthoredPublications = await getCoAuthoredPublications(id);
+    
     const where: Prisma.PublicationWhereInput = {
-        createdBy: id,
+        OR: [
+            { createdBy: id },
+            { id: {
```

Comment:
This can be simplified by querying publications where user is part of co-authors list.
```
  OR: [
            { createdBy: id },
            {
                coAuthors: {
                    some: {
                        linkedUser: id
                    }
                }
            }
        ],
        currentStatus: {
            in: statuses
        }
```

Then, there's no need for any additional helper service like: 'coAuthoredPublications';
There are some eslint/prettier complains about formatting.